### ISSUE TITLE
Extra `'NoneType` in py2many print test case

### ISSUE THREAD

**Issue by jayvdb** [Coronon/PySchemeTranspiler#7]
https://github.com/adsharma/py2many/blob/d5ef9c3/tests/cases/print.py (after removing the `if __name__ == "__main__"`) transpiles to 
```racket
#lang racket
(define (PRINT . args) (for-each (lambda (x) (display x) (display " ")) args)(newline))
(define (!= a b) (if (and (number? a) (number? b)) (not (= a b)) (not (equal? a b))))

(define (show ) (PRINT "b")(PRINT 2 "b")(define a 2.1)(PRINT a)(define b 2.1)(PRINT b)(define c #t)(PRINT (if c "True" "False"))'NoneType)
(cond (#t (show)))
```
which emits roughly the same as the Python code, except an extra `'NoneType` at the end.

Also related, PySchemeTranspiler is emitting `b \n` instead of `b\n` in its translation of the Python `print("b")`.

**Comment by Coronon** [comment]
This is actually not a fault of my implementation but rather the default behaviour of racket. All top level expressions are printed out if not either `(void)` or `(values)`. This means that a `(define a (someFunc))` will not print anything but `(someFunc)` will if *someFunc* actually returns something. This _could_ be solved by wrapping everything in a:
```racket
#lang racket

(define (main)
    ;...
    (void)
)
(main)
```
Would that help you or would you consider that *ugly* (like me ^^ <- Im not opposed to the idea though if that gets us closer to the Python behaviour)

**Comment by jayvdb** [comment]
If it gets us closer to Python behaviour, then it sounds necessary.
You might also like to activate this only when encountering `if __name__ == "__main__":`

**Comment by Coronon** [comment]
Fixed in 291d05322dc9fb1593fea8b32c328c898f88d2f8

I did not go with detecting `if __name__ == "__main__":` as this is probably quite useful to everyone.

Btw. would it be helpful for you if I actually internally assign `__name__` so that this lookup always works? Racket can not simply be imported anyways.
